NuGet.Services.Dashboard.FrontEnd/NuGet.Services.Dashboard.FrontEnd.csproj

```diff
@@ -449,6 +499,10 @@
       <Project>{5566ae9c-8466-4316-9372-7b38b57dbb20}</Project>
       <Name>NuGet.Services.Dashboard.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\NuGet.Services.Dashboard.Operations\NuGet.Services.Dashboard.Operations.csproj">
```

Comment:
>ices.Dashboard.Operations\NuGet.Services.Dashboard.Operations [](start = 44, length = 61)

please don't add a reference to the Operations project in the Front_end project. Those were separated intentionally. Any common code should be in  Dashboard.Common